DependencyInjection/SmartboxIntegrationFrameworkExtension.php

```diff
@@ -43,14 +47,15 @@ class SmartboxIntegrationFrameworkExtension extends Extension
     const CONSUMER_PREFIX = 'smartesb.consumers.';
     const PARAM_DEFERRED_EVENTS_URI = 'smartesb.uris.deferred_events';
     const AMQP_SERVICES = [
-        'smartesb.amqp.queue_manager',
-        'smartesb.consumers.async_queue',
         'smartesb.drivers.queue.amqp',
+        'smartesb.async_consumers.queue'
     ];
 
     protected $config;
     protected $useAmqp = false;
 
+    protected $drivesThatUseAMQP = [];
```

Comment:
This property can be deleted because is not needed

Reply:
Done.